test/python/visualization/test_pass_manager_drawer.py

```diff
@@ -52,7 +52,8 @@ def setUp(self):
         self.pass_manager.append(Unroller(basis_gates))
         self.pass_manager.append(CheckMap(coupling_map))
         self.pass_manager.append(BarrierBeforeFinalMeasurements(), do_while=lambda x: False)
-        self.pass_manager.append(CXDirection(coupling_map))
+        with self.assertWarns(DeprecationWarning):
+            self.pass_manager.append(CXDirection(coupling_map))
```

Comment:
I'm not sure why this is now necessary. The warning should have been triggered before when creating `CXDirection`.